frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/ReviewInformation/EditCamperCard.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useMemo, useState } from "react";
+import {
+  Box,
+  Button,
+  Divider,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  Heading,
+  Input,
+  Spacer,
+  Text,
+  Textarea,
+  Wrap,
+  WrapItem,
+} from "@chakra-ui/react";
+import {
+  PersonalInfoActions,
+  PersonalInfoReducerDispatch,
+} from "../../../../../../types/PersonalInfoTypes";
+import { RegistrantExperienceCamper } from "../../../../../../types/CamperTypes";
+import {
+  checkAge,
+  checkFirstName,
+  checkLastName,
+} from "../../PersonalInfo/personalInfoReducer";
+import { CampResponse } from "../../../../../../types/CampsTypes";
+import EditFormLabel from "./EditFormLabel";
+import EditCardFooter from "./EditCardFooter";
+
+type EditCamperCardProps = {
+  camper: RegistrantExperienceCamper;
+  camperIndex: number;
+  camp: CampResponse;
+  dispatchPersonalInfoAction: (action: PersonalInfoReducerDispatch) => void;
+};
+
+const EditCamperCard = ({
+  camper,
+  camperIndex,
+  dispatchPersonalInfoAction,
+  camp,
+}: EditCamperCardProps): React.ReactElement => {
+  const [updateMemo, setUpdateMemo] = useState(0);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const initialCamper: RegistrantExperienceCamper = useMemo(() => camper, [
+    updateMemo,
+  ]);
+
+  const [editing, setEditing] = useState(false);
+
+  const [isFirstNameInvalid, setIsFirstNameInvalid] = useState<boolean>(false);
+  const [isLastNameInvalid, setIsLastNameInvalid] = useState<boolean>(false);
+  const [isAgeInvalid, setIsAgeInvalid] = useState<boolean>(false);
```

Comment:
for consistency I would recommend just doing `useState(false)` I think the <boolean> can be omitted

Reply:
Done.